web/src/validators/social-media-urls.js

```diff
@@ -0,0 +1,18 @@
+import { helpers } from 'vuelidate/lib/validators';
+
+const linkedinUrl = (value) => {
```

Comment:
Wow, I was thinking just checking the domain name, but you really went the extra mile. Cool 😎